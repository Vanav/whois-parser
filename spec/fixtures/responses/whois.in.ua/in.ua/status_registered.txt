*
* Whois query server SUNIC
*

domain:             dle.in.ua
registrar:          ua.nic
registrant:         vp535-sunic
admin:              vp535-sunic
tech:               nic-sunic
nserver:            ns10.uadns.com
nserver:            ns12.uadns.com
nserver:            ns11.uadns.com
status:             ok
created:            2007-12-18 20:12:35
updated:            2017-12-19 12:54:31
expires:            2018-12-20 00:00:00
source:             SUNIC

Contact nic-sunic
--------------------
id:                 nic-sunic
email:              support@nic.ua
phone:              +380.445933222
fax:                +380.445937569
addr_int:           not published
name_int:           NIC-UANIC
org_int:            NIC.UA LLC

Contact vp535-sunic
--------------------
id:                 vp535-sunic
email:              not published
phone:              not published
fax:                not published
addr_loc:           not published
name_loc:           not published
org_loc:            not published

