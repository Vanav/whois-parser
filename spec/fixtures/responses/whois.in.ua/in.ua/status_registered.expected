#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2007-12-18 17:12:35}

#updated_on
  %s %CLASS{time}
  %s %TIME{2017-12-19 09:54:31}

#expires_on
  %s %CLASS{time}
  %s %TIME{2018-12-19 21:00:00}


#nameservers
  %s %CLASS{array}
  %s %SIZE{3}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns10.uadns.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns12.uadns.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns11.uadns.com"
